components/wise/actions/get-balance/get-balance.mjs

```diff
@@ -0,0 +1,40 @@
+import wise from "../../wise.app.mjs";
+
+export default {
+  name: "Get Balance",
+  version: "0.0.1",
+  key: "wise-get-balance",
+  description: "Get a list of balance. [See docs here](https://api-docs.wise.com/api-reference/balance#get)",
```

Comment:
A list of balances is provided in the prop, and the action only returns data about one specific balance, right?

Perhaps this description could reflect that, e.g. `Get details for a balance` since it does not actually return a list